src/auth/types/jwtPayload.ts

```diff
@@ -0,0 +1,4 @@
+export type JwtPayload = {
+	email: string;
+	sub: string;
+};
```

Comment:
自動付与されるなら不要


exp(期限を示すもの)も欲しいかも
```suggestion
	exp: number;
};
```